app-interface/manifests/rhose-fleet-manager.yml

```diff
@@ -256,11 +256,11 @@ objects:
                     secretKeyRef:
                       name: ${FLEET_MANAGER_SECRET_NAME}
                       key: SHARD_ROUTER_CANONICAL_HOSTNAME
-                - name: EVENT_BRIDGE_DNS_HOSTED_ZONE_ID
+                - name: EVENT_BRIDGE_DNS_HOSTED_ZONE_NAME
                   valueFrom:
                     secretKeyRef:
                       name: ${FLEET_MANAGER_SECRET_NAME}
-                      key: EVENT_BRIDGE_DNS_HOSTED_ZONE_ID
+                      key: EVENT_BRIDGE_DNS_HOSTED_ZONE_NAME
```

Comment:
Does this mean you'd need to deploy a new version of our secrets too?

Reply:
I've already added the new `EVENT_BRIDGE_DNS_HOSTED_ZONE_NAME` secret on aws sm. I will remove `EVENT_BRIDGE_DNS_HOSTED_ZONE_ID` when this gets deployed to the envs